hackclub.com.yaml

```diff
@@ -1,4327 +1,4485 @@
+- octodns:
+    cloudflare:
+      proxied: true
+  ttl: 300
+  type: CNAME
+  value: a.selfhosted.hackclub.com.
 slushies:
-  - ttl: 600
-    type: CNAME
-    value: a.selfhosted.hackclub.com.
+- octodns:
+    cloudflare:
+      proxied: true
+  ttl: 300
+  type: CNAME
+  value: a.selfhosted.hackclub.com.
 tile:
-  - ttl: 600
-    type: CNAME
-    value: a.selfhosted.hackclub.com.
+- octodns:
+    cloudflare:
+      proxied: true
+  ttl: 300
+  type: CNAME
+  value: a.selfhosted.hackclub.com.
 touch:
-  - ttl: 600
-    type: CNAME
-    value: a.selfhosted.hackclub.com.
+- octodns:
+    cloudflare:
+      proxied: true
+  ttl: 300
+  type: CNAME
+  value: a.selfhosted.hackclub.com.
 touch-grass:
-  - ttl: 600
-    type: CNAME
-    value: a.selfhosted.hackclub.com.
+- octodns:
+    cloudflare:
+      proxied: true
+  ttl: 300
+  type: CNAME
+  value: a.selfhosted.hackclub.com.
 infinity:
-  - ttl: 600
-    type: CNAME
-    value: a.selfhosted.hackclub.com.
-ows: #outernet-web-services
-  - ttl: 600
-    type: NS
-    values:
-      - ns1.he.net.
-      - ns2.he.net.
-      - ns3.he.net.
-      - ns4.he.net.
-      - ns5.he.net.
+- octodns:
+    cloudflare:
+      proxied: true
+  ttl: 300
+  type: CNAME
+  value: a.selfhosted.hackclub.com.
+ows:
+- ttl: 300
+  type: NS
+  values:
+  - ns1.he.net.
+  - ns2.he.net.
+  - ns3.he.net.
+  - ns4.he.net.
+  - ns5.he.net.
 p2p:
-  ttl: 600
+  ttl: 300
   type: CNAME
   value: p2phackclub.github.io.
 pa:
-  - ttl: 600
-    type: CNAME
-    value: cname.vercel-dns.com.
+- ttl: 300
+  type: CNAME
+  value: cname.vercel-dns.com.
 pack:
-  ttl: 600
+  ttl: 300
   type: CNAME
   value: cname.vercel-dns.com.
 packages:
-  ttl: 600
+  ttl: 300
   type: CNAME
   value: secret-zebra-lydr194iy8x0x4n4rmzcii3r.herokudns.com.
 panorama:
-  ttl: 600
+  ttl: 300
   type: CNAME
   value: panorama-countdown.pages.dev.
 parcel-pyxis:
-  - ttl: 600
-    type: CNAME
-    value: a.selfhosted.hackclub.com.
-parthenon: 
-  - ttl: 600
-    type: CNAME
-    value: d5aba33cc6b279e0.vercel-dns-017.com.
+- octodns:
+    cloudflare:
+      proxied: true
+  ttl: 300
+  type: CNAME
+  value: a.selfhosted.hackclub.com.
+parthenon:
+- ttl: 300
+  type: CNAME
+  value: d5aba33cc6b279e0.vercel-dns-017.com.
 passport:
-  ttl: 600
+  ttl: 300
   type: CNAME
   value: cname.vercel-dns.com.
 pathfinder:
-  ttl: 600
+  ttl: 300
   type: CNAME
   value: meghanam4.github.io.
 pfp:
-  - ttl: 600
-    type: A
-    value: 77.90.16.198
+- ttl: 300
+  type: A
+  value: 77.90.16.198
 philanthropy:
-  ttl: 600
+  ttl: 300
   type: CNAME
   value: cname.vercel-dns.com.
 philippines:
-  - ttl: 600
-    type: CNAME
-    value: cname.vercel-dns.com.
+- ttl: 300
+  type: CNAME
+  value: cname.vercel-dns.com.
 phs:
-  - ttl: 600
-    type: ALIAS
-    value: cname.vercel-dns.com.
-  - ttl: 600
-    type: MX
-    values:
-      - exchange: mx1.improvmx.com.
-        preference: 1
-      - exchange: mx2.improvmx.com.
-        preference: 5
```

Comment:
This MX record should not have been deleted! This is a breaking change, was this communicated to the team responsible?